src/aqa.ts

```diff
@@ -73,23 +74,47 @@ async function run(): Promise<void> {
     if (sdkdir === '') {
       sdkdir = process.cwd()
     }
-    await runaqa.runaqaTest(
-      version,
-      jdksource,
-      customizedSdkUrl,
-      sdkdir,
-      buildList,
-      target,
-      customTarget,
-      aqatestsRepo,
-      openj9Repo,
-      tkgRepo,
-      vendorTestParams,
-      aqasystemtestsRepo
-    )
+    if(runParallel === 'true') {
```

Comment:
Could we enhance this condition to check the numMachines in case that runParalllel is set true and numMachines is set to 1?